src/server/index.ts

```diff
@@ -89,4 +91,11 @@ server
   // For all others paths
   .get("*", fallbackHandler);
 
+if (config.hsClientId === "ecency.app" && defaults.base === "https://ecency.com" && config.hsClientSecret.length === 0) {
```

Comment:
what's the purpose of these lines, I assume it was for testing purpose?!

Reply:
The lines are to prevent invalid configurations from frustrating running and malfunctioning.  It is better to fail and close.  In the first branch, this is the default setting.  Once this case is taken into account any of these four settings will be an invalid configuration.

Reply:
Hmm, I think, it doesn't necessarily serve the purpose. Perhaps trying like, warning message respond is better?! Check here: `src/server/util.tsx`. @shawnpringle